src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs

```diff
@@ -86,32 +86,33 @@ internal class SafeGssCredHandle : SafeHandle
         /// </summary>
         public static SafeGssCredHandle Create(string username, string password, string domain)
         {
-            SafeGssCredHandle retHandle = null;
```

Comment:
The diffs would have been easier to read if you simply change the init of retHandle :) thusly:
````
SafeGssCredHandle retHandle = new SafeGssCredHandle; // invalid handle
````